services/spar/spar.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.33.0.
+-- This file has been generated from package.yaml by hpack version 0.34.2.
```

Comment:
This *shouldn't* cause any problems, but if it does, you need to find the right stack version to match the one on our other systems (concourse, mostly, I think?).

But I think it's fine.

Reply:
Hm, this is not ideal - we can upgrade stack, but then we should upgrade everywhere at the same time, and also update CI (best done in a separate PR). Could you revert your version of stack to  `2.3.1` and re-compile spar / re-generate this file?

Reply:
I reverted to stack 2.3.1 and regenerated it. (Don't know if it's relevant, but I noticed stack provided by the `shell.nix` in the root of `wire-server` is `2.3.3`.)